.gitattributes

```diff
@@ -0,0 +1 @@
+flow/flow-api-client/** linguist-generated=true
```

Comment:
Can exclude README.md from the gitattributes to make sure it pops up in the changes

Reply:
Thanks! Updated in [2dd052d](https://github.com/PeerDB-io/peerdb/pull/1996/commits/2dd052dde9a65086270501ba1c134169bd3ee3a2)